crates/wash-cli/src/bin/wash.rs

```diff
@@ -98,19 +99,8 @@ Options:
   -V, --version          Print version
 ";
 
-/// Helper function to display the version of all the binaries wash runs
-fn version() -> String {
-    format!(
-        "         v{}\n├ nats-server {}\n├ wadm        {}\n└ wasmcloud   {}",
-        clap::crate_version!(),
-        NATS_SERVER_VERSION,
-        WADM_VERSION,
-        WASMCLOUD_HOST_VERSION
-    )
-}
-
 #[derive(Debug, Clone, Parser)]
-#[clap(name = "wash", version = version(), override_help = HELP)]
+#[clap(name = "wash", version, override_help = HELP)]
```

Comment:
Sorry @ericgregory looks like this reverted a few changes 😢 this may have been a rebase issue

Reply:
Changes should be restored in the latest commit!